apps/web_delivery/mix.exs

```diff
@@ -36,4 +31,11 @@ defmodule WebDelivery.Mixfile do
       {:rollbax, "~> 0.8.0"},
     ]
   end
+
+  defp elixirc_paths(:test), do: ["lib", "web", "test/support"]
+  defp elixirc_paths(_),     do: ["lib", "web"]
+
+  def application do
+    [mod: {WebDelivery.Application, []}, extra_applications: [:logger]]
+  end
```

Comment:
Ok, that makes sense. It is just going to take a little time to get used to. I'm used to putting all public functions first in alphabetical order and then all private functions last in alphabetical order.